cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -442,7 +442,10 @@ pGovernanceCmd :: Parser GovernanceCmd
 pGovernanceCmd =
   Opt.subparser $
     mconcat
-      [ Opt.command "create-update-proposal"
+      [ Opt.command "create-MIR-certificate"
+          (Opt.info pMIRCertificate $
+            Opt.progDesc "Create an MIR certificate")
```

Comment:
Perhaps we should expand on the "MIR" abbreviation:

```suggestion
            Opt.progDesc "Create an MIR (Move Instantaneous Rewards) certificate")
```

Reply:
might as well yes